scripts/interective/menu.rb

```diff
@@ -134,20 +137,29 @@ def self.tests_list(prompt)
     clear
     puts 'Checks: testing'
     choices = [
-      { name: 'Test search [rake test:search]', value: 1 },
-      { name: 'Test observe [rake test:observe]', value: 2 },
-      { name: 'Test cancel [rake test:cancel]', value: 3 },
-      { name: 'back', value: 4 },
-      { name: 'exit', value: 5 }
+      { name: 'Test binding search [rake test:binding:search]', value: 1 },
+      { name: 'Test binding observe [rake test:binding:observe]', value: 2 },
+      { name: 'Test binding cancel [rake test:binding:cancel]', value: 3 },
+      { name: 'Test matcher nodejs [rake test:matcher:nodejs]', value: 4 },
+      { name: 'Test matcher rust [rake test:matcher:rust', value: 5 },
+      { name: 'Test all [rake test:all', value: 6 },
+      { name: 'back', value: 7 },
+      { name: 'exit', value: 8 }
     ]
     case prompt.select('Actions groups', choices)
     when 1
-      Rake::Task['test:search'].invoke
+      Rake::Task['test:binding:search'].invoke
     when 2
-      Rake::Task['test:observe'].invoke
+      Rake::Task['test:binding:observe'].invoke
     when 3
-      Rake::Task['test:cancel'].invoke
+      Rake::Task['test:binding:cancel'].invoke
     when 4
+      Rake::Task['test:matcher:nodejs'].invoke
+    when 5
```

Comment:
maybe name it `karma` instead `nodejs`? What do you think, it isn't really nodejs... I don't know - depend on you